draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -685,6 +697,13 @@ are dispatched during the ICE candidate gathering and processing described in
 ensure a single origin does not prevent other origins from registering,
 unregistering, or resolving mDNS names.
 
+mDNS queries from outside the local link
+----------------------------------------
+
+The browser is expected to answer to mDNS query messages, which might have the
+response-unicast bit set. In such a case, it is important to follow {{RFC6762}}
+section 5.5 and ignore mDNS query messages originating from outside the local link.
```

Comment:
```suggestion
Section 5.5 and ignore mDNS query messages originating from outside the local link.
```

Reply:
Is there ever a case where we would receive a direct unicast query as described in S 5.5? I tend to think that we might just want to disallow this altogether, and avoid the complexities of subnet matching.

Reply:
I think the attack @tQsW mentions is not really direct unicast queries (mDNS spec already deals with it).
AIUI, there is regular multicast query with unicast-response bit set and the source is set to outside the local link, which can lead to potential DOS attacks. I'll remove the reference to section 5.5 which seems misleading.